gating/generate_release_notes/generate_release_notes.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -xe
+
+# This script is run within a docker container to generate release notes.
+
+url=$(git remote -v |awk '/origin.*fetch/{print $2}')
+rpc-differ --debug -r "$url" --update "$PREVIOUS_TAG" "$NEW_TAG" --file notes.rst
+pandoc --from rst --to markdown_github < notes.rst > notes.md
```

Comment:
What happens in the case where there are no release notes? How do we deal with that?

Reply:
Rpc_differ can generate a report between identical versions, it just isn't very interesting.